index.html

```diff
@@ -14,14 +14,17 @@
 <body>
         <header class="main-header"> 
             <a class="logo" href="#"><img src="images/My Logo (1).png" alt="home page logo"></a>
-            <nav class="nav-container">
+            <nav id="nav-container">
+                    <a class="desk" href="#" >Project</a>
+                    <a class="desk" href="#" >About</a> 
+                    <a class="desk" href="#" >Contact</a>
```

Comment:
Figma
![image](https://user-images.githubusercontent.com/78251927/229274849-c3c8a0fe-d1c4-4c63-be00-30aedd3b4d02.png)

Your design
![image](https://user-images.githubusercontent.com/78251927/229274868-a5207f15-9c6c-41db-a6dc-d09ef15f8041.png)

Please kindly change the text `color` and `weight` to match the template. 👍

Reply:
I have observed that you forgot to implement the suggestion provided by the previous reviewer @abdulhamiid